### ISSUE TITLE
Intel Arc Control Fan Curve suddenly absent; Arc Control competes with Predator Bifrost App for fan control

### ISSUE THREAD

**Issue by gargamel314** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#851]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Intel Arc Control; Predator Bifrost App

### Processor / Processor Number [Required]

i7-13700K

### Graphic Card [Required]

Intel Arc/Acer Predator Bifrost A770 16GB

### GPU Driver Version [Required]

Other (Please specify)

### Other GPU Driver version

32.0.101.6078 (and prior releases)

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build [Required]

Windows 11 23H2

### Intel System Support Utility report

[SSU - Intel Arc Control (2).txt](https://github.com/user-attachments/files/17179598/SSU.-.Intel.Arc.Control.2.txt)

### Description and steps to reproduce [Required]

For context, I've been trying to set a fan curve in  Arc Control. because the automatic fan curve is set too conservatively, and doesn't ramp up to 100% until the card is already near 90F under full load. The Acer Predator Bifrost app only has "Automatic" and "Manual." When I would set this control to 100%, it would go to 100%, for a few moments, and then suddenly decrease and seemingly revert back to following the automatic fan curve. It seems like the Predator BiFrost App and Arc Control are competing for control over the fans. Because of this unreliable behavior, I decided to disable the Predator Bifrost App from starting with Windows and control the A770 strictly with Intel Arc Control.

Sometimes, the custom fan curve I set in arc control works perfectly with all the results I expect. Sometimes, Arc control will show that the fan curve is set, but it seems to follow the automatic fan curve. Sometimes, the Performance Tuning Configure page will revert to default settings upon reboot, even though I have selected "Apply settings on system boot" to "on," and then reselecting the fan curve will help. Sometimes, upon reboot, I will look at the Performance Tuning page, and the fan control slider is completely missing, as shown in the screenshot provided. The fan appears to follow the Automatic fan curve in that situation. 

I do have Fan Control V200 (https://getfancontrol.com/) installed, with an Intel Arc GPU die sensor plugin installed, but this *only* shows sensor data, it does not have any control the fan. I use it simply to sense the A770's temps and turn on case fans when the temps get too high.

![Screenshot 2024-09-29 105332](https://github.com/user-attachments/assets/908efffd-cd82-4553-aa4b-8fe12132a4a5)

### Device / Platform

ASUS Strix Z690-A D4

### Crash dumps [Required, if applicable]

no crashes

**Comment by Felipe-Intel** [comment]
Hello @gargamel314 sorry to not have answered before. Looks like we have two issues here. 
Issue 1: Acer app competing with Arc Control
Issue 2: Fans not behaving correctly with custom fan control settings, also (looks like) reverting to default. 
Is this correct? Please let me know if I understood correctly your report so we can tackle each problem appropriately.  
Also, you mention "fan curve suddenly absent" in the title but I don't see anything related to it in the issue description. Could you please elaborate on this? Thanks!

**Comment by gargamel314** [comment]
> Is this correct? Please let me know if I understood correctly your report so we can tackle each problem appropriately.
Yes, that covers it. Acer app is competing with Arc Control specifically for fan control.

> Also, you mention "fan curve suddenly absent" in the title but I don't see anything related to it in the issue description. Could you please elaborate on this? Thanks!

This happens very intermittently. Suddenly upon boot, You open Arc Control, click the Performance tab, then click Performance Tuning. Coincidentally as I'm typing this response, it has done it again. I'll add a screenshot. Notice that the Fan Control option is gone (Automatic, Curve, Manual), and there is no slider or graph to set fan speed or curve.  Earlier today, this showed as normal, showing the fan curve graph and option. 

![Screenshot 2024-10-03 215433](https://github.com/user-attachments/assets/5029d3dc-760e-476c-9d10-9a7473ef9964)

**Comment by Felipe-Intel** [comment]
I didn't realize the screenshot you posted originally showed the issue. Got it. Then: 

Issue 1: Acer app competing with Arc Control
Issue 2: Fans not behaving correctly with custom fan control settings, also (looks like) reverting to default.
Issue 3: Fan curve absent

I will be working on this three things and let you know my findings.

I noticed in the SSU that you have Parsec app installed and a dedicated sound card. Any other detail you think important? I am suspecting of Parsec and how Arc Control service handles changes in display devices (we had an issue with Parsec in the past, already fixed). If possible, could you please disable or uninstall Parsec and check if issues 2 and 3 happens?

**Comment by gargamel314** [comment]
sure, I will disable Parsec. It may take a few days until I have results.

**Comment by Felipe-Intel** [comment]
No problem. That gives me time to try to replicate on my side.

**Comment by Felipe-Intel** [comment]
@gargamel314 quick update: 
Issue 1, Acer app competing with Arc Control: Under investigation
Issue 2, Fans not behaving correctly with custom fan control settings, also (looks like) reverting to default: Not able to reproduce. I set up the manual control for Bifrost app and as long as I didn't open ArcC, fan speed was constant. Then, close/disable Bifrost app and set up fan curve in ArcC. After multiple reboots the settings remained the same and fan speed matched the setting. Didn't seem to go to default. If you have more details on these, please share.
Issue 3: Fan curve absent: Under investigation.

**Comment by ImposterProfessorOak** [comment]
I experience an issue similar to this. Apologies if there's extra or some iffy overlap with other problems as I will try to be thorough.

I own an A770 8GB, ASRock variant. I also have the issue of the fan control intermittently going away. Getting to this state typically happens on my end after waking the device (from sleep/hibernation), after a reboot, from a driver update, or so on. Coincidentally, the fans will also ramp up to a very high speed after a certain temp and that lets me know the "bug" has occurred.

Some key oddities I can note in this state: the control for the fans is missing, the performance monitoring for the fans is also missing from Arc Control and its overlay. Any software to check the fan performance will show it is either inaccessible or remain at 0rpm regardless of actual fan performance.

In this state, I can corroborate that the GPU will follow the "automatic" setting. After a reboot, the fans will quiet down, despite there being no user accessible fan control. With GPU load increasing the temps, the fans will kick into the max setting instantly with no ramp-up.

Sometimes, a few reboots here and there will bring the fan and its controls back to being accessible. Unfortunately, I just leave my PC on more or less nowadays to avoid this problem from occurring. I have no legitimate way of replicating this, other than it seemingly occurring at random at the listed instances (wake from sleep/hibernation/after reboot/after driver update).

Simultaneously, just to add my experiences in, BiFrost and Arc Control do coexist in my situation. Just the Arc card and its control will sometimes go away for no truly recognizable reason.

**Comment by gargamel314** [comment]
Disabling Parsec has not helped - 4 or 5 times last week the fan control option and slider disappeared

**Comment by Felipe-Intel** [comment]
Noted. This is still under investigation so I will add your comment to the case.

**Comment by Felipe-Intel** [comment]
Hello! @gargamel314 @ImposterProfessorOak we have published a few new drivers since this issue was posted. Specifically, the versions of Arc Control from driver 6078 and current 6130 are different. Are you still experiencing the loss of the fan speed control option with this new version? I would recommend to first go to add/remove programs and look for "Intel(R) Arc Software & Drivers" and uninstall everything, then run DDU,  and finally install the new version selecting the option of clean install. 

Regarding the "competing apps", it was confirmed by our ArcControl team that third party software applications may cause conflicts when leveraging similar APIs to adjust overlapping or matching settings (in this particular case, the fan speed settings). If installing multiple software applications that have control over the same feature enablement, we recommend being mindful not to adjust settings that may cause conflicts. This includes configuring settings in more than one application to be automatically applied at start or boot time. This is not exclusive to Arc Control and the BiFrost app, but to any two or more apps that can control the same setting in a PC, whatever this setting is.

**Comment by gargamel314** [comment]
I will try these recommendations and report back. Thank you.

**Comment by Felipe-Intel** [comment]
@gargamel314 @ImposterProfessorOak following up with you to see if you were able to test the newest driver. Yesterday we released driver [32.0.101.6297](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) . You can try with this one.

**Comment by ImposterProfessorOak** [comment]
Howdy!

New driver install on the latest 6297. DDU and everything. Initially, the fan speed monitoring was missing in the performance graphs and monitoring. However, I was now able to set a custom curve AND the monitoring returned! A good first step, as usually the control is lost if it doesn't appear the first time.

I'll be giving this the usual routine now, as I stopped sleeping or hibernating the PC in fear of the fans losing control. Should I be able to recreate my issues, I'll consider creating a separate ticket as this is gargamel314's ticket initially. 👍

**Comment by Felipe-Intel** [comment]
> Howdy!
> 
> New driver install on the latest 6297. DDU and everything. Initially, the fan speed monitoring was missing in the performance graphs and monitoring. However, I was now able to set a custom curve AND the monitoring returned! A good first step, as usually the control is lost if it doesn't appear the first time.
> 
> I'll be giving this the usual routine now, as I stopped sleeping or hibernating the PC in fear of the fans losing control. Should I be able to recreate my issues, I'll consider creating a separate ticket as this is gargamel314's ticket initially. 👍

Noted. If find again the problem (let's hope not), it is ok to create a new ticket but please make reference to this one.

**Comment by Felipe-Intel** [comment]
hello @gargamel314 following up with you to see if you were able to test the newest driver. Yesterday we released driver [32.0.101.6299](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) . You can try with this one.

**Comment by gargamel314** [comment]
@Felipe-Intel 

Yes, I've tested it. Fan control gone today, it's the first time I've noticed it, but here's a screenshot. I'm on 6299, Yesterday I had a fan curve defined and had flipped on "Apply settings on boot." It still forgets my settings even if I flip that toggle, and now the fan control is gone again. 

I did as you suggested. Acer Predator app is gone from my system and I haven't had any competing fan controls. 

![Driver version 6299 Scrnsht](https://github.com/user-attachments/assets/84bebd35-3128-4924-89be-7b4ffdf15f38)

**Comment by Felipe-Intel** [comment]
Hello @gargamel314 as you may have noticed, we replace Arc Control software with the new Intel Graphics Software, included in our driver [101.6325](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html)
Would you mind checking if this behavior persists on the new software?

**Comment by ImposterProfessorOak** [comment]
> Hello @gargamel314 as you may have noticed, we replace Arc Control software with the new Intel Graphics Software, included in our driver [101.6325](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) Would you mind checking if this behavior persists on the new software?

Thank you for the notification of a change in software. Coincidentally, I lost fan control today after waking from sleep. Unfortunately, I will have to reinstall the driver and software to regain control but I will try to write up a different bug report for my issue specifically if it returns on the newest software set.

**Comment by gargamel314** [comment]
> Hello @gargamel314 as you may have noticed, we replace Arc Control software with the new Intel Graphics Software, included in our driver [101.6325](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) Would you mind checking if this behavior persists on the new software?

Hi there, I did install the new drivers, and have been playing with the new Intel Graphics Control software. I haven't seen the fan control slider disappear, HOWEVER, when you set a fan curve, upon reboot that fan curve resets to the default every time, while the changes to other performance settings stay set through reboots.

**Comment by Felipe-Intel** [comment]
@gargamel314 Noted. Let's keep this thread open for the original issue so you can confirm the issue is gone in IGS. As for the other, please go ahead an create a new post so we can follow up that there. I'll test in our lab and let you know.

**Comment by Felipe-Intel** [comment]
@gargamel314 I was able to repro the fan settings not persisting after reboot issue. Have you created a new ticket? Also let me know if/when you are ok with closing this ticket (i.e. fan curve disappearing).

**Comment by Felipe-Intel** [comment]
@gargamel314 Last week we released a [new driver](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) with a new version of IGS. I see the fan settings persisting. Could you please confirm? if that is the case, I think this ticket can be closed. Looking forward for your answer.

**Comment by gargamel314** [comment]
Hi there. yes i have noticed the fan settings staying the same on reboot. I also have not seen the fan slider disappearing as I had before. I'm sorry, I have been very busy with my job lately and haven't had a lot of time for testing.
